scripts/zfs.sh

```diff
@@ -44,12 +46,14 @@ while getopts 'hvud' OPTION; do
 		exit 1
 		;;
 	v)
+		# shellcheck disable=SC2034
 		VERBOSE=1
 		;;
 	u)
 		UNLOAD=1
 		;;
 	d)
+		# shellcheck disable=SC2034
```

Comment:
This really is unused dead functionality.  The spl logging facility has been removed, this and the spl_dump_log() function should go.